examples/python/ColumnOperations/ExtensionService_column.py

```diff
@@ -113,6 +117,42 @@ def _sum_of_column(request):
         # Yield the row data constructed
         yield SSE.BundledRows(rows=[SSE.Row(duals=duals)])
 
+
+    
+
+    @staticmethod
+    def _max_of_columns_2(request, context):
+        """
+        Find max of each column. This is a table function.
+        :param request: an iterable sequence of RowData
+        :param context:
+        :return: a table with numerical values, two columns and one row
+        """
+
+        result = [_MINFLOAT]*2
+
+        # Iterate over bundled rows
+        for request_rows in request:
+            # Iterating over rows
+            for row in request_rows.rows:
+                # Retrieve numerical value of parameter and append to the params variable
+                # Length of param is 1 since one column is received, the [0] collects the first value in the list
```

Comment:
This comment is a bit confusing, is 'the params variable' the result? And 'Length of param is 1 since one column is received' kind of implies that the function only expects one column but the function returns two columns...

Reply:
In earlier examples/versions of the example, 'row' has been called 'param'. Updating the comment must have been missed in the change. I can change that as well.

Reply:
Okej :). The code is included in one .md document as well, you might want to change that one too.

Should it be length of row is 2 then?

Reply:
Thanks for pointing that out, I'll change it there as well. I plan to change the documentation for the python examples to avoid this issue with duplicated information. We'll see when I have the time to do that.

Yes in this case, 'row' will be of length 2.